src/App/App.js

```diff
@@ -1,12 +1,25 @@
-import React from 'react';
-import './App.css';
+import React, { useState } from 'react';
+import './App.scss';
 
+import unsplash from '../helpers/unsplash'
 import Header from '../Header/Header'
 
 
 
 
+
 const App = () => {
+  const [images, setImages] = useState();
+
```

Comment:
Search function to get back ten photos from unsplash, haven't fully committed to allowing the user to search or have a dropdown to narrow the results for MVP.